src/declarativedbusinterface.cpp

```diff
@@ -351,7 +352,7 @@ void DeclarativeDBusInterface::typedCall(const QString &method, const QJSValue &
     QDBusPendingCallWatcher *watcher = new QDBusPendingCallWatcher(pendingCall);
     connect(watcher, SIGNAL(finished(QDBusPendingCallWatcher*)),
             this, SLOT(pendingCallFinished(QDBusPendingCallWatcher*)));
-    m_pendingCalls.insert(watcher, callback);
+    m_pendingCalls.insert(watcher, qMakePair(callback, errorCallback));
```

Comment:
I wouldn't consider that an error. Actually that would be even an API break, suddenly everyone needs to add also error callback. Now it's an extra developers can opt-in for.